### DISCUSSION TITLE
oUF layout

### DISCUSSION THREAD

**Discussion by tidehunter1** [p3lim-wow/LibEditMode#21] [Questions]
i'm learning oUF, how would i go about adding this to a oUF layout?

**Comment by p3lim** [comment]
There's already [oUF MovableFrames](https://github.com/oUF-wow/oUF_MovableFrames), which you could contribute to or fork, replacing the positioning system with LEM.

**Comment by tidehunter1** [comment]
Thanks for the reply, I just figured the LEM was more straightforward to the user expecting to see movable frames in the edit mode.
I added the code provided in the wiki straight to the oUF:Factory
```
oUF:Factory(function(self)
local player = self:Spawn("player", "oUF_Player")
LEM:AddFrame(player, onPositionChanged, defaultPosition)
...
```
it mostly works, frames move etc except "layoutName" returns nil and i havn't been able to figure out how to resolve it.
